dali/pipeline/executor/executor.h

```diff
@@ -113,6 +113,11 @@ class DLL_PUBLIC Executor : public ExecutorBase, public WorkspacePolicy, public
   DLL_PUBLIC void ReleaseOutputs() override;
   DLL_PUBLIC void SetCompletionCallback(ExecutorCallback cb) override;
 
+  DLL_PUBLIC void ShutdownQueue() {
+    QueuePolicy::ReleaseOutputIdxs();
```

Comment:
Shouldn't we release all the indexes we hold? So a `in_use_queue_.size()` calls here?
As we already are being destructed, we can release everything.

We can also not care and just `SignalStop()`, we can also consider moving it to the top of this function?

Reply:
Done